src/porting.h

```diff
@@ -360,6 +360,9 @@ inline u32 getDeltaMs(u32 old_time_ms, u32 new_time_ms)
 	}
 #elif defined(_WIN32)
 	inline void setThreadName(const char* name) {}
+#elif defined(__GNU__)
+//#warning "GNU/Hurd platform, pthread_setname_np not yet supported"
+inline void setThreadName(const char* name) {}
```

Comment:
On 07.02.2015 18:43, nerzhul wrote:
> In src/porting.h
> <https://github.com/minetest/minetest/pull/2164#discussion_r24291042>:
> 
>> @@ -360,6 +360,9 @@ inline u32 getDeltaMs(u32 old_time_ms, u32 new_time_ms)
>>  	}
>>  #elif defined(_WIN32)
>>  	inline void setThreadName(const char* name) {}
>> +#elif defined(__GNU__)
>> +//#warning "GNU/Hurd platform, pthread_setname_np not yet supported"
>> +inline void setThreadName(const char* name) {}
> 
> Agree with @ShadowNinja <https://github.com/ShadowNinja>
> 

Please tell me exactly how you want this to be fixed since I am not the
original patch creator. You can also fix this part in any way you see fit.